scripts/images/convert_qcow2_ova.py

```diff
@@ -9,22 +9,32 @@
 import urllib.request
 import tempfile
 import shutil
+import stat
 from urllib.parse import urlparse
 from pathlib import Path
 from jinja2 import Template
 
 
 
-help_message = """convert_qcow2_ova.py -u <imageUrl> -s <imageSize> -n <imageName>
+help_message = """convert_qcow2_ova.py -u <imageUrl> -s <imageSize> -n <imageName> -d <imageDist> -U <rhUser> -P <rhPassword> -O <osPassword>
 
--u, --imageUrl                 URL pointing to the qcow2.gz rhcos image
+-u, --imageUrl                 URL pointing to the qcow2.gz or the absolute local file path
 -s, --imageSize                Size of the image you want to create
 -n, --imageName                Name of the ova image file
+-d, --imageDist                Image distribution; coreos, rhel
```

Comment:
I'm wondering if we can just make it more generic so that we can use it for the centos as well if possible!

Reply:
or is it even possible to have support for centos? if yes - then we can look into adding the support for the same in next PR or so?

Reply:
it should work for CentOS also, but yet to test it